app/controllers/tags_controller.rb

```diff
@@ -10,4 +12,27 @@ def index
       format.json { render :json => @tags }
     end
   end
+
+  def new
+    @tag = Tag.new
+    @title = "Create Tag"
+  end
+
+  def create
+    tag = Tag.create!(params.require(:tag).permit(:tag, :description, :privileged, :is_media, :inactive, :hotness_mod))
```

Comment:
Tag needs to validate the non-empty presence of the `tag` field, the `tags` table should be migrated to remove the `""` default, and this action and `update` need to check that validations passed. I'm not really OK with the console letting you break things in such easily prevented ways, and a web interface definitely shouldn't allow it.